docker/cygnus-ngsi/Dockerfile

```diff
@@ -340,7 +333,7 @@ RUN \
     find /usr/share/locale -mindepth 1 -maxdepth 1 ! -name 'en_US' ! -name 'locale.alias' | xargs -r rm -r && rm -f /var/log/*log && \
     bash -c 'localedef --list-archive | grep -v -e "en_US" | xargs localedef --delete-from-archive' && \
     /bin/cp -f /usr/lib/locale/locale-archive /usr/lib/locale/locale-archive.tmpl && \
-    #build-locale-archive && find ${CYGNUS_HOME} -name '.[^.]*' 2>/dev/null | xargs -r rm -rf && \
+    #build-locale-archive && find ${CYGNUS_HOME} -name '.[^.]*' 2>/dev/null | xa1rgs -r rm -rf && \
```

Comment:
xa1rgs is a typo?

Reply:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/2225/commits/b404fe4ba92b76dd041a709ac45a8a622bf309c6